examples/20-calendar.raku

```diff
@@ -0,0 +1,46 @@
+
+use GTK::Simple;
+use GTK::Simple::App;
+
+my $app = GTK::Simple::App.new(title => 'Calendar', height => 300, width => 600);
+my $calendar = GTK::Simple::Calendar.new;
+
+my $month-entry = GTK::Simple::Entry.new(text => ~$calendar.month);
+my $year-entry  = GTK::Simple::Entry.new(text => ~$calendar.year);
+my $day-entry   = GTK::Simple::Entry.new(text => ~$calendar.day);
+
+$calendar.day-selected.tap: {
+    $year-entry.text    = .year.Str;
+    $month-entry.text   = .month.Str;
+    $day-entry.text     = .day.Str;
+};
+
+my $date-view = GTK::Simple::Grid.new(
+    [0, 0, 1, 1] => GTK::Simple::Label.new(text => "Day"),
+    [1, 0, 1, 1] => $day-entry,
+    [2, 0, 1, 1,] => GTK::Simple::Label.new(text => "Month"),
+    [3, 0, 1, 1] => $month-entry,
+    [4, 0, 1, 1] => GTK::Simple::Label.new(text => "Year"),
+    [5, 0, 1, 1] => $year-entry
+);
+
+$date-view.column-spacing = 8;
+
+my $structure = GTK::Simple::HBox.new(
```

Comment:
I think you should Grid instead of either HBox or VBox. See - https://github.com/raku-community-modules/gtk-simple/issues/27

Reply:
Will fix, thanks!

Reply:
Actually, considering all of the existing examples which still rely on this, I'd prefer to address it in a separate comprehensive PR.

Reply:
Well, I decided.. why not both? :grin: